bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/MiIoDevices.java

```diff
@@ -184,6 +184,17 @@ public enum MiIoDevices {
     ROBOROCK_VACUUM_A15("roborock.vacuum.a15", "Roborock S7", THING_TYPE_VACUUM),
     ROBOROCK_VACUUM_A19("roborock.vacuum.a19", "Roborock S4 Max", THING_TYPE_VACUUM),
     ROBOROCK_VACUUM_A23("roborock.vacuum.a23", "Roborock T7S Plus", THING_TYPE_VACUUM),
+    ROBOROCK_VACUUM_A26("roborock.vacuum.a26", "Roborock G10S Pro", THING_TYPE_VACUUM),
+    ROBOROCK_VACUUM_A27("roborock.vacuum.a27", "Roborock S7 MaxV", THING_TYPE_VACUUM),
+    ROBOROCK_VACUUM_A29("roborock.vacuum.a29", "Roborock G10", THING_TYPE_VACUUM),
+    ROBOROCK_VACUUM_A30("roborock.vacuum.a30", "Roborock G10", THING_TYPE_VACUUM),
+    ROBOROCK_VACUUM_A34("roborock.vacuum.a34", "Roborock Q5", THING_TYPE_VACUUM),
+    ROBOROCK_VACUUM_A37("roborock.vacuum.a37", "Roborock T8", THING_TYPE_VACUUM),
+    ROBOROCK_VACUUM_A38("roborock.vacuum.a38", "Roborock Q7 Max", THING_TYPE_VACUUM),
```

Comment:
Please see #12993 (changes and comments). It seems some additional changes are needed when adding new models.

Reply:
Why do you think that?
Pr https://github.com/openhab/openhab-addons/pull/12993 tried to add it as basic device. In that case indeed other additions are needed.
Based on the input from the forum I don't think this device needs to be added as miio:basic instead as miio:vacuum as done here.

Reply:
OK. I'm not into this binding, so I hope someone can step in and assist with review and knowledge. Perhaps @cribskip can add some context having created the former PR? For example, why roborock.vacuum.a38-miot.json was added. There is also this comment I tried to point to: https://github.com/openhab/openhab-addons/pull/12993#issuecomment-1164030146 - so seems like at least the README should be updated, and that there is a tool for that.

Reply:
Former PR also updated basic.properties.